Makefile

```
@@ -30,7 +30,7 @@ generate:
 
 .PHONY: build
 build: generate
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o otel-go-instrumentation cli/main.go
+	GOOS=linux GOARCH=amd64 go build -o otel-go-instrumentation cli/main.go
```

Could detect architecture using `uname` so generate will work for both amd and arm:

```suggestion
	GOOS=linux GOARCH=$(uname -m) go build -o otel-go-instrumentation cli/main.go
```